scripts/actions/mobskills/self-destruct.lua

```diff
@@ -4,7 +4,10 @@
 local mobskillObject = {}
 
 mobskillObject.onMobSkillCheck = function(target, mob, skill)
-    if mob:isMobType(xi.mobskills.mobType.NOTORIOUS) or mob:getHPP() > 75 then
+    if
+        (mob:isMobType(xi.mobskills.mobType.NOTORIOUS) or mob:getHPP() > 75) and
+        mob:getFamily() ~= 91 -- Pandemonium Lamp can always use
```

Comment:
Suggested to use pool id instead of family id. The lamps -should- already be in their own pool making that safe to do (I've been slowly trying to eradicate family checks, and thank you for getting the others out of the mobskill scripts).

Reply:
Switched to pool ID check